tests/kafkatest/services/security/security_config.py

```diff
@@ -172,6 +173,52 @@ def kafka_opts(self):
         else:
             return ""
 
+    def set_acls(self, protocol, kafka, zk, topic, group):
```

Comment:
Fair comment from @granders. I need to rebase based on a merge I did yesterday so I'll pull it out now.